test/ad/utils.jl

```diff
@@ -11,19 +11,21 @@ struct DistSpec{VF<:VariateForm,VS<:ValueSupport,F,T,X,G}
     x::X
     "Transformation of sample `x`."
     xtrans::G
+    "Broken backends"
+    broken::Tuple{Vararg{String}}
```

Comment:
That's basically what I thought we could do. However, I thought one could use symbols instead of strings and maybe add a type parameter (i.e., `broken::B` where `B <: Tuple`).

Reply:
Good idea!